README.md

```diff
@@ -131,6 +131,8 @@ This project wouldn't be possible without our awesome contributors. Thank you >:
             <a href="https://github.com/swarup4741"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/swarup4741.jpg" width="120px;" alt="Swarup Kumar Das"/><br /><sub><b>Swarup Kumar Das</b></sub></a></td>
         <td align="center">
             <a href="https://www.facebook.com/melnic.ira/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/iramelnic.jpg" width="120px;" alt="Ira Melnic"/><br /><sub><b>Ira Melnic</b></sub></a></td>
+        <td align="center">
+            <a href="https://github.com/lazarnarcis"><img src="https://avatars.githubusercontent.com/u/82839799?v=4" width="120px;" alt="Narcis Lazar"/><br /><sub><b>Narcis Lazar</b></sub></a></td>
```

Comment:
You need to add the image inside our repo, and reference it from there. Look at how the rest of the images are added